tests/store/artifact/test_azure_data_lake_artifact_repo.py

```diff
@@ -0,0 +1,237 @@
+import os
+import posixpath
+import pytest
+from unittest import mock
+
+from azure.storage.filedatalake import (
+    DataLakeServiceClient,
+    FileSystemClient,
+    DataLakeDirectoryClient,
+    DataLakeFileClient,
+)
+from azure.storage.filedatalake import PathProperties
+from mlflow.exceptions import MlflowException
+from mlflow.store.artifact.azure_data_lake_artifact_repo import (
+    AzureDataLakeArtifactRepository,
+    _parse_abfss_uri,
+)
+
+
+TEST_ROOT_PATH = "some/path"
+TEST_DATA_LAKE_URI_BASE = "abfss://filesystem@account.dfs.core.windows.net"
+TEST_DATA_LAKE_URI = posixpath.join(TEST_DATA_LAKE_URI_BASE, "some/path")
```

Comment:
```suggestion
TEST_DATA_LAKE_URI = posixpath.join(TEST_DATA_LAKE_URI_BASE, TEST_ROOT_PATH)
```

Can we use `TEST_ROOT_PATH`?